packages/analytics-js-service-worker/src/types.ts

```diff
@@ -12,7 +12,9 @@ export interface ApiObject {
     | undefined
     | ApiObject
     | unknown
```

Comment:
`Unknown` doesn't make sense here. 
ESLint must have also thrown a warning. Isn't it?

Reply:
Yes, addressed